diff --git a/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs b/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
index 64e6666..cca591f 100644
--- a/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
+++ b/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
@@ -382,7 +382,9 @@ var ajvOrig = (additionalOptions = {}) => {
 
     ajv.addMetaSchema(metaSchema);
     // eslint-disable-next-line no-underscore-dangle
-    ajv._opts.defaultMeta = metaSchema.id;
+    if (ajv._opts) {
+        ajv._opts.defaultMeta = metaSchema.id;
+    }
 
     return ajv;
 };
diff --git a/node_modules/@eslint/eslintrc/dist/eslintrc.cjs b/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
index eb2ed8d..4446d01 100644
--- a/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
+++ b/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
@@ -1623,7 +1623,9 @@ var ajvOrig = (additionalOptions = {}) => {
 
     ajv.addMetaSchema(metaSchema);
     // eslint-disable-next-line no-underscore-dangle
-    ajv._opts.defaultMeta = metaSchema.id;
+    if (ajv._opts) {
+        ajv._opts.defaultMeta = metaSchema.id;
+    }
 
     return ajv;
 };
